docs/conf.py

```diff
@@ -102,6 +102,22 @@ def django_release():
     'ticket': ('https://code.djangoproject.com/ticket/%s', '#'),
 }
 
+# Load django apps for generating their source by 'sphinx.ext.viewcode'
+try:
+    import django
+    from django.conf import settings
+except ImportError:
+    pass
+else:
+    settings.configure()
+    settings.INSTALLED_APPS = [
+        'django.contrib.admin',
+        'django.contrib.auth',
+        'django.contrib.contenttypes',
+        'django.contrib.sessions',
+    ]
+    django.setup()
```

Comment:
I don't think that we need to call `django.setup()`. I would prefer to load only necessary apps, i.e.
```python
try:
    from django.conf import settings
    from django.apps import apps
except ImportError:
    pass
else:
    settings.configure()
    apps.populate(['django.contrib.contenttypes', 'django.contrib.auth'])

```

Reply:
@felixxm Thanks for the review.
As I [commented](https://code.djangoproject.com/ticket/29942#comment:3) on the ticket. there is also some missed source page from `session` and `admin` apps.
If we don't populate these apps we will miss the following source files:

- django/contrib/admin/models.html
- django/contrib/admin/apps.html
- django/contrib/sessions/base_session.html

I don't know the exact location of the source link for these there files, I found them by making the diff between files with your patch and my old patch.

If you want to know the exact location of them I can find them.